crates/host/src/wasmbus/mod.rs

```diff
@@ -3651,6 +3802,66 @@ impl Host {
         Ok(ACCEPTED.into())
     }
 
+    #[instrument(skip(self, data))]
```

Comment:
Debug? I would also consider `skip_all` and putting the fields on the log line. It renders more clearly

Reply:
This is something we should maybe decide on, but I personally don't like manually setting fields every time I need them. If there is something I think should be on every log line for context, I don't want to manually put it in there

Reply:
We synced offline and agreed in theory putting the field son the span makes more sense, but the formatter we're using makes the logs hard to read that way 😢

Reply:
Leaving this one unresolved because it's still instrumented at an info level

Reply:
Should be done now